specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2024-05-01-preview/appplatform.json

```diff
@@ -11079,6 +11205,45 @@
       },
       "x-ms-discriminator-value": "Weekly"
     },
+    "ConfigServerResourceRequests": {
+      "description": "Resource request payload of Config Server, only available in Enterprise Plan.",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "cpu": {
+          "description": "CPU allocated to each Config Server instance",
+          "type": "string",
+          "readOnly": true
+        },
+        "memory": {
+          "description": "Memory allocated to each Config Server instance",
+          "type": "string",
+          "readOnly": true
+        },
+        "instanceCount": {
```

Comment:
Do you want to put it to SKU like other resource?

Reply:
Currently we do not support auto scale for config server.